.github/workflows/create-release.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: create-release
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Conventional Changelog Action
```

Comment:
Is this going to commit directly on `main`? Will it be able to?

Reply:
It should commit to main yes, I've been testing it in a personal repo which has the same permissions set and it is able to. 

As to why/how, since it has 

```
permissions:
  contents: write
```

that allows the action to create a release (from [here](https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs#overview). So I _think_ its ability to push a commit to `main` is with the restriction that it's a release PR and not introducing any additional code. There's a separate (currently disabled) permission that would allow an Action to be able to merge PRs using the GitHub token. 

Speaking of, I think I can remove the permission: pull requests since that's not needed here